core/src/snapshot_package.rs

```diff
@@ -240,6 +280,15 @@ mod tests {
         // Make tarball from packageable snapshot
         SnapshotPackagerService::package_snapshots(&snapshot_package).unwrap();
 
+        // before we compare, stick an empty status_cache in this dir so that the package comparision works
```

Comment:
Copypasta?

Reply:
Yeah, it was in test code in two different modules and I didn't want to move it into the compare function. Decided to leave it as is since it's not something that should be overlooked.